tests/auto/tst_persistenttabmodel/tst_persistenttabmodel.cpp

```diff
@@ -428,11 +494,45 @@ void tst_persistenttabmodel::onTitleChanged()
 
     QSignalSpy dataChangedSpy(tabModel, SIGNAL(dataChanged(QModelIndex, QModelIndex, QVector<int>)));
 
-    mockPage.m_tabId = 1;
-    mockPage.setTitle("Hello world");
+    EXPECT_CALL(mockPage, tabId()).WillOnce(Return(1));
+    QUrl url;
+    EXPECT_CALL(mockPage, url()).WillOnce(Return(url));
+    EXPECT_CALL(mockPage, title()).WillOnce(Return(QString("Hello world")));
+    emit mockPage.titleChanged();
+
     QCOMPARE(dataChangedSpy.count(), 1);
 }
 
+void tst_persistenttabmodel::nextActiveTabIndex()
+{
+    DeclarativeWebContainer container;
+    DeclarativeWebPage page;
+    tabModel->setWebContainer(&container);
+
+    EXPECT_CALL(container, webPage()).WillRepeatedly(Return(&page));
+    EXPECT_CALL(page, parentId()).WillOnce(Return(1));
+    EXPECT_CALL(page, tabId()).WillOnce(Return(2));
+    tabModel->nextActiveTabIndex(0);
+}
+
+void tst_persistenttabmodel::misc()
+{
+    // Call simple methods to increase coverage
```

Comment:
Increasing coverage by calling just random functions doesn't make much sense if there's nothing really tested.

For instance roleNames() coverage improvement could be done with a ListView + PersistentTabModel QML test and using valid and invalid roles in the delegate. ListView::currentItem in turn could be used to check that data is read to the correctly by using role names.